### ISSUE TITLE
DL1 columns

### ISSUE THREAD

**Issue by MaxNoe** [cta-observatory/cta-lstchain#343]
Looking at the DL1 Parameter files, there are some columns I wouldn't expect at that location in the 
`dl1/event/telescope/parameters/telescope` section.

These are:

MC information (should go in `/sim/.../`?
```
mc_alt
mc_alt_tel
mc_az
mc_az_tel
mc_core_distance
mc_core_x
mc_core_y
mc_energy
mc_h_first_int
mc_type
mc_x_max
log_mc_energy
```

Disp: are these truth values or already predictions? Why are RF predictions already inside this? 
```
disp_angle
disp_dx
disp_dy
disp_norm
disp_sign
```

Telescope positions:
```
tel_pos_x
tel_pos_y
tel_pos_z
```

**Comment by vuillaut** [comment]
Hi.

Yes, the data model is clearly not right, it was just easier to handle at the beginning.
I meant to update it but then the DL1 writer was expected along with the DL1 data model so it will get updated at this point.

`disp_***` are truth values. `mc_` was dedicated to info coming directly from the simtel file and in the MC container, hence the difference.
`reco_***` are reconstructed values.

**Comment by rlopezcoto** [comment]
@FrancaCassol was also discussing with me yesterday about some of the fields in this structure. I agree with @vuillaut that we should update it whenever the DL1 writer is out

**Comment by MaxNoe** [comment]
Yes, I concur. This should be part of ctapipe 0.8 and can be done with the effort of adapting that
